components/kubeflow/kfserving/src/kfservingdeployer.py

```diff
@@ -429,12 +429,21 @@ def update(kfsvc, model_name, namespace):
                 exit(1)
     try:
         print(
-            model_status["status"]["url"] + " is the knative domain."
+            model_status["status"]["address"]["url"] + " is the knative domain."
+        )
+        print("Sample test commands: ")
```

Comment:
can you remove line 434-439, and line 444? thanks

Reply:
as you wish